.github/workflows/publish-operator-docker-image.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         with:
           push: true
           file: Dockerfile
-          platforms: linux/amd64
+          platforms: linux/amd64,linux/arm64
```

Comment:
For sure, it is not the only required change.

arm64 packages are avaialbe in [splunk-opentelemetry-dotnet-linux-glibc-arm64.zip](https://github.com/signalfx/splunk-otel-dotnet/releases/download/v1.5.0/splunk-opentelemetry-dotnet-linux-glibc-arm64.zip) and [splunk-opentelemetry-dotnet-linux-musl-arm64.zip](https://github.com/signalfx/splunk-otel-dotnet/releases/download/v1.5.0/splunk-opentelemetry-dotnet-linux-musl-arm64.zip)

Dockerfile is not referencing them: https://github.com/signalfx/splunk-otel-dotnet/blob/f6ad54d9f078f650374d9aff2c216487043a3480/Dockerfile#L23-L24

What is more, it will be great to start this work on https://github.com/open-telemetry/opentelemetry-operator and then propagate changes on our side.